hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/DeletedBlockLogImpl.java

```diff
@@ -270,15 +300,8 @@ private boolean isTransactionFailed(DeleteBlockTransactionResult result) {
   @Override
   public void addTransaction(long containerID, List<Long> blocks)
       throws IOException {
-    lock.lock();
-    try {
-      Long nextTXID = scmMetadataStore.getNextDeleteBlockTXID();
-      DeletedBlocksTransaction tx =
-          constructNewTransaction(nextTXID, containerID, blocks);
-      scmMetadataStore.getDeletedBlocksTXTable().put(nextTXID, tx);
-    } finally {
-      lock.unlock();
-    }
+    Map<Long, List<Long>> map = Collections.singletonMap(containerID, blocks);
+    addTransactions(map);
```

Comment:
How about remove this `addTransaction()`, since it is only used in unit test ?

Reply:
This can be done in my DeleteBlock pr.